libbeat/cfgfile/reload.go

```diff
@@ -167,49 +167,7 @@ func (rl *Reloader) Run(runnerFactory RunnerFactory) {
 
 			debugf("Number of module configs found: %v", len(configs))
 
-			startList := map[uint64]Runner{}
-			stopList := rl.registry.CopyList()
-
-			for _, c := range configs {
-
-				// Only add configs to startList which are enabled
-				if !c.Enabled() {
-					continue
-				}
-
-				rawCfg := map[string]interface{}{}
-				err := c.Unpack(rawCfg)
-
-				if err != nil {
-					logp.Err("Unable to unpack config file due to error: %v", err)
-					continue
-				}
-
-				hash, err := hashstructure.Hash(rawCfg, nil)
-				if err != nil {
-					// Make sure the next run also updates because some runners were not properly loaded
```

Comment:
Is this case still handled?

Reply:
Nope :/ I'll have a look and find a solution

Reply:
See 4ea21f2